docs/source/api/recipes/recipes.ipynb

```diff
@@ -85,7 +85,7 @@
    "outputs": [
```

Comment:
<p>"length" not "lenght", "an" not "and", "remove length-0 paths" not "remove lengths 0". The last sentence is a bit confusing to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>